src/vast_tracker.js

```diff
@@ -178,35 +180,39 @@ export class VASTTracker extends EventEmitter {
     }
 
     if (this.assetDuration > 0) {
-      const events = [];
-
-      if (progress > 0) {
-        const percent = Math.round((progress / this.assetDuration) * 100);
-
-        events.push('start');
-        events.push(`progress-${percent}%`);
-        events.push(`progress-${Math.round(progress)}`);
-
-        for (const quartile in this.quartiles) {
-          if (
-            this.isQuartileReached(quartile, this.quartiles[quartile], progress)
-          ) {
-            events.push(quartile);
-            this._alreadyTriggeredQuartiles[quartile] = true;
+      const percent = Math.round((progress / this.assetDuration) * 100);
+      for (let i = this.lastPercentage; i < percent; i++) {
+        const events = [];
+        if (progress > 0) {
+          events.push('start');
+          events.push(`progress-${i + 1}%`);
+          events.push(`progress-${Math.round(progress)}`);
```

Comment:
As the for loop only concern the percentage progress, we should only push in the events array the missing progress-percentage (`events.push('progress-${i + 1}%');`). Here, we might trigger multiple time 'start' or `progress-${Math.round(progress)}` with the same value, also their is no need to calculate the quartiles inside of it. 
 We need to keep thing same as before but just add your for loop that will push in the events array the missing progress-percentage.